.gitmodules

```diff
@@ -351,3 +351,6 @@
 [submodule "contrib/aklomp-base64"]
 	path = contrib/aklomp-base64
 	url = https://github.com/aklomp/base64.git
+[submodule "contrib/pocketfft"]
+	path = contrib/pocketfft
+	url = https://gitlab.mpcdf.mpg.de/mtr/pocketfft.git
```

Comment:
Looks like this repo is pretty much dead, and that development moved on to https://github.com/mreineck/pocketfft. Let's reference the other repo.